egs/wsj/s5/steps/make_mfcc.sh

```diff
@@ -151,6 +151,12 @@ if $write_utt2num_frames; then
   rm $logdir/utt2num_frames.*
 fi
 
+# Store frame_shift and mfcc_config along with feats.scp.
+frame_shift=$(perl -ne 'if (/^--frame-shift=(.*?)([\s#]|$)/) {
+                          printf "%.3f", 0.001 * $1; exit; }' $mfcc_config)
+echo ${frame_shift:-'0.01'} > $data/frame_shift
+cp $mfcc_config $data/mfcc.conf
```

Comment:
If we are to store the config I'd prefer to put it in $data/conf/{mfcc,plp,mfcc_pitch}.conf, or whatever.  We should keep the scripts in sync (although we rarely use theothers).

Reply:
> If we are to store the config I'd prefer to put it in $data/conf/{mfcc,plp,mfcc_pitch}.conf, or whatever.

Will do.

> We should keep the scripts in sync (although we rarely use the others).

Sure, I wanted to run the approach by you first, this is why this change is only to one of them. So, are all 3 changes ok by you: 1) create utt2num_frames by default (this is what you suggested, so I guess it is); 2) pull frame_shift from config, or store the default; and 3) store the configs with the feature dataset (but in a conf/ subdirectory)?